build_scripts/windows/Product.wxs

```diff
@@ -61,7 +61,7 @@
 
   <Fragment>
     <Directory Id="TARGETDIR" Name="SourceDir">
-      <Directory Id="ProgramFilesFolder">
+      <Directory Id="ProgramFiles64Folder">
```

Comment:
If you're going to dual-build x86 and x64 MSIs, there's a different pattern you need to use for this. I have examples in PowerShell, OpenSSH, et. al.